BAC0/core/io/Read.py

```diff
@@ -414,32 +421,36 @@ def readMultiple(
                 raise UnknownObjectError("Unknown object {}".format(args))
             elif reason == "unknownProperty":
                 self._log.warning("Unknown property {}".format(args))
-
-                values.append("")
+                values.append("")  # type: ignore[arg-type]
```

Comment:
I'm not sure about adding an empty string to the values list but a couple other code paths were doing it so I left it intact

Reply:
I think it's because I found no other good way to fill a list of values with one missing...

Reply:
ahh ok, well that makes sense at least. and yeah I agree with you I don't think there's a better way without writing up a whole class to handle the various `read` return types